app/build.gradle

```diff
@@ -160,13 +160,13 @@ dependencies {
   testImplementation "com.google.truth:truth:$versions.truth"
```

Comment:
Accidentally approved these changes without testing this PR. No idea how to "unapprove" so starting another review here.